README.md

```diff
@@ -32,6 +32,14 @@ If true, adds a random delay (10s-30s) before joining a meeting. Can be useful s
 - **auto_leave_after_min:**  
 If set to a value greater than zero, the bot leaves every meeting after the specified time (in minutes). Useful if you know the length of your meeting, if this is left a the default the bot will stay in the meeting until a new one is available.
 
+- **additional_acc:** 
+if u have addititonal account in same mail or anything similar to given pic 
+<img width="80%" alt="jarray reverse exampl" src="https://i.imgur.com/VPmecz2.png">
+change additional_acc by which no of account u want to select "counting start from 1 here"
```

Comment:
What you mention here as accounts are actually organisations, this has to be changed everywehere.
```suggestion
- **organisation_num:** 
If your Teams account is in multiple organisations, as seen in the example below, change the organisation_num to the number of the list item (counting starts from 1) 
<img width="80%" alt="jarray reverse exampl" src="https://i.imgur.com/VPmecz2.png">
```